libensemble/gen_classes/gpCAM.py

```diff
@@ -38,32 +35,31 @@ class GP_CAM(LibensembleGenerator):
     (relative to the simulation evaluation time) for some use cases.
     """
 
-    def _initialize_gpCAM(self, user_specs):
-        """Extract user params"""
-        # self.b = user_specs["batch_size"]
-        self.lb = np.array(user_specs["lb"])
-        self.ub = np.array(user_specs["ub"])
+    def __init__(self, VOCS: VOCS, ask_max_iter: int = 10, random_seed: int = 1, *args, **kwargs):
+
+        super().__init__(VOCS, *args, **kwargs)
+        self.rng = np.random.default_rng(random_seed)
+
+        self._validate_vocs(VOCS)
```

Comment:
this is at top level right?

Reply:
Good point; I made it the first thing the superclass's init does